soci/soci_index.go

```diff
@@ -65,7 +63,7 @@ var (
 	errUnsupportedLayerFormat = errors.New("unsupported layer format")
 )
 
-// Index represents an ORAS/OCI Artifact Manifest
+// Index represents an ORAS/OCI Artifact Manifest.
```

Comment:
Can remove `ORAS` here.

Reply:
This comment will be updated by #356. Revert my change to avoid conflict